setup.py

```diff
@@ -39,7 +39,7 @@ def read(fname):
     keywords=['socorro', 'breakpad', 'crash', 'reporting', 'minidump',
               'stacktrace'],
     packages=find_packages(),
-    install_requires=[], # use peep -r requirements.txt instead
+    install_requires=[], # use pip -r requirements.txt instead
```

Comment:
For my packages, requirements.txt represents a known-good set of dependencies: the set I've tested with and the one I deploy with. It uses only the == comparator. setup.py's install_requires is a more permissive set of versions, generally using > and <. It's targeted more at people importing my project as a part of their own, for which they might need different exact versions of deps.